.migration-backup/ui-components/src/services/class/__tests__/access.test.ts

```
@@ -0,0 +1,220 @@
+import { ClassAccessControl } from '../access';
+import { supabase } from '../../../utils/supabase';
```

ESLint

[error] '../../../utils/supabase' import is restricted from being used by a pattern. Avoid deep relative imports. Use path aliases (@ui, @domains, @lib, etc.) instead.

(no-restricted-imports)